### ISSUE TITLE
Frontend layout manager and particle editor

### ISSUE THREAD

**Issue by AmibeWebsites** [gantry/gantry5#2073]
When?

I've mentioned this many times (https://github.com/gantry/gantry5/issues/1499#issuecomment-239433432, https://github.com/gantry/gantry5/issues/1499#issuecomment-250103262, https://github.com/gantry/gantry5/issues/974#issuecomment-252545995 and https://github.com/gantry/gantry5/issues/2022#issuecomment-310598536) and this is hard for me to write because I love Gantry and have invested a lot of time and attention into using it. All the websites I've build use Gantry and I have been planning to based my entire service offering on Gantry.

I would love to build a frontend app that interacts with the Gantry API but do not have to time, resources or technical knowledge at this stage. For me to reliably get those resources I need to demonstrate the power of frontend editing. A catch-22.

I understand if it's not a priority and must then accept that Gantry is not the right tool for what I'm trying to achieve. Regardless of my excitement for Gantry, my users simply don't care how great it is if they cannot drag-and-drop elements around their site like (and I quote them all here) "I can do in Wix".

I read an article recently about Beaver Builder, which spoke about 2nd Gen page builders being those that provide front-end editing. I took a look at Divi Builder and was impressed by the frontend user experience. I can't help but wonder why I can't do this with Gantry.

I know, I know... Gantry is a theme framework and not a page builder... yet these page builders are doing what I feel a theme framework should also. And while Divi doesn't (yet) offer the single most compelling feature of Gantry, assignment of layouts, there is another free page builder that does (just not nearly as well as Gantry does).

It's a strange concept to wrap ones head around but in my very humble opinion themes are dead. I'm planning a talk for an upcoming WordCamp where I'll demonstrate a no-theme theme, creating an entire site design on the frontend. I would have loved to have been demonstrating Gantry.

**Comment by Pixelpaule** [comment]
Yes that's right. For me Gantry is also more than just a template framework and I also miss the opportunity to edit the front-end with a drag and drop editor. Are there any plans to implimentade this in gantry?
